batch/batch/front_end/templates/openapi.yaml

```diff
@@ -1445,5 +1457,690 @@ paths:
               schema:
                 type: object
 
-#components:
-  # TODO: Add response types
+components:
+  schemas:
+    BillingLimitRequest:
+      type: object
+      required:
+        - limit
+      properties:
+        limit:
+          type: number
+          description: "The billing limit value in dollars. Use null to remove the limit."
+          nullable: true
+          example: 1000.50
+
+    CreateBatchRequest:
+      type: object
+      required:
+        - billing_project
+        - attributes
+      properties:
+        billing_project:
+          type: string
+          description: "The billing project to charge for this batch"
+        attributes:
+          type: object
+          description: "Key-value pairs of attributes for the batch"
+          additionalProperties:
+            type: string
+        callback:
+          type: string
+          description: "Optional callback URL to notify when batch completes"
+        token:
+          type: string
+          description: "Optional token for batch identification"
```

Comment:
These tokens are actually mixed-in by the python code. They're not API parameters